pdl/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/pdl/LocalCache.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.pdl
+
+import java.time.LocalDateTime
+import kotlin.time.Duration
+import kotlin.time.toJavaDuration
+
+class LocalCache<T>(
```

Comment:
Generelt OK, kunne også vare REDIS. Vi ville ha ekstra latensitet på netverket , men likevel vile spare tid på grunn av binar formaten og det at det ikke brukes REST og andre systemer. Jeg persjonlig vile kanskje gå for redis, men dette er også OK.